test-case/check-pause-release-suspend-resume.sh

```diff
@@ -121,6 +121,10 @@ rm -rf /tmp/sof-test.lock
 #   catch: Evaluate script and trap exceptional returns
 #   after ms: Ms must be an integer giving a time in milliseconds.
 #       The command sleeps for ms milliseconds and then returns.
+# FIXME: this test was broken by 46dadd0 ("Add mutual exclusion and journalctl logging")
+#       because it never was compatible with is_subtest()
+# FIXME: Need to handle more of aplay/arecord output. Need to apply similar fix with
+#       check-pause-resume.sh, multiple-pause-resume.sh
```

Comment:
Why volume is MAX is separate question

No, that's why this fails in the first test failed in the first place.
If the ALSA settings are not correct, this PR is not useful.

Fix the ALSA settings first, then we can refine expect scripts.

Reply:
> Fix the ALSA settings first, then we can refine expect scripts.

As usual, keep the broken ALSA settings in your backpocket to "test the test" locally. A test that can't fail is useless (green failure).